metadata-ingestion/README.md

```diff
@@ -401,6 +402,25 @@ source:
     # options is same as above
 ```
 
+### Feast `feast`
+
+Extracts:
+
+- List of feature tables (modeled as `MLFeatureTable`s), features (`MLFeature`s), and entities (`MLPrimaryKey`s)
+- Column types associated with each feature and entity
+
+Note: this uses a separate Docker container to extract Feast's metadata into a JSON file, which is then
+parsed to DataHub's native objects. This was done because of a dependency conflict in the `feast` module.
```

Comment:
Does that mean the host running this must have docker? If so, may be useful to call out such reqs here also

Reply:
Yes, will add this as a requirement